docs/framework/wcf/extending/configuration-and-metadata-support.md

```diff
@@ -176,7 +176,7 @@ if (transportBindingElement is UdpTransportBindingElement)
  The custom binding element can export policy assertions in the WSDL binding for a service endpoint to express the capabilities of that binding element. The following example code is taken from the [Transport: UDP](../../../../docs/framework/wcf/samples/transport-udp.md) sample.  
   
 #### Policy Export  
- The `UdpTransportBindingElement` type implements``<xref:System.ServiceModel.Description.IPolicyExportExtension?displayProperty=nameWithType> to add support for exporting policy. As a result, <xref:System.ServiceModel.Description.MetadataExporter?displayProperty=nameWithType> includes `UdpTransportBindingElement` in the generation of policy for any binding that includes it.  
+ The `UdpTransportBindingElement` type implements `<xref:System.ServiceModel.Description.IPolicyExportExtension?displayProperty=nameWithType>` to add support for exporting policy. As a result, `<xref:System.ServiceModel.Description.MetadataExporter?displayProperty=nameWithType>` includes `UdpTransportBindingElement` in the generation of policy for any binding that includes it.  
```

Comment:
The correct fix would be to remove the code fence character on all the xrefs. The xref will style the element correctly.